bowling-ball/Game.cs

```diff
@@ -2,16 +2,63 @@
 
 namespace BowlingBall
 {
+    
     public class Game
     {
-        public void Roll(int pins)
-        {
-            throw new NotImplementedException();
+        private int[] Rolls = new int[21];

```

Comment:
As per naming conventions, private variables should start with an underscore and follow camel casing.